MyOtherClass.cls

```diff
@@ -1,11 +1,13 @@
-public without sharing class MyClass {
+public with sharing class MyOtherClass {
```

Comment:
Class `MyOtherClass` does not enforce sharing correctly. Please change to `inherited sharing`.